### PR TITLE
Update gfx

### PR DESCRIPTION
**Pull request by parasyte**
- Fixes the screen resolution and mouse coordinates on macOS
- Fixes the blurry font by using linear filtering on the texture sampler
- Fixes the default color scheme when using gfx

### CODE REVIEW

**Comment by parasyte** [comment]
The second patch addresses #54

**Comment by Gekkio** [comment]
Thanks! :smile:

File: imgui-examples/examples/support_gfx/mod.rs [hunk 1]
```diff
@@ -58,7 +57,7 @@ pub fn run<F: FnMut(&Ui) -> bool>(title: String, clear_color: [f32; 4], mut run_
     let mut mouse_state = MouseState::default();
     let mut quit = false;
 
-    loop {
+    'running: loop {
```
Comment (kyrylo) [inline imgui-examples/examples/support_gfx/mod.rs]: May I ask you guys the implications of using this? I thought these labels were designed for nested loops. Is it necessary to have it here?

Comment (parasyte) [reply imgui-examples/examples/support_gfx/mod.rs]: @kyrylo A leftover from an attempted refactor that didn't work out. The label can be removed.

Comment (malikolivier) [reply imgui-examples/examples/support_gfx/mod.rs]: Thanks! I opened a merge request addressing what you guys reported: https://github.com/Gekkio/imgui-rs/pull/145

**Pull request merged**